spec/support/feature_spec_helper.rb

```diff
@@ -1,6 +1,4 @@
 module FeatureSpecHelper
-  # requires IntegrationSpecHelper
```

Comment:
We have a lot of spec files that require each other and don't make a comment about each one. Why is this file any different?